docs/src/pages/_document.page.tsx

```diff
@@ -36,9 +36,9 @@ const getCSPContent = (context: Readonly<HtmlProps>) => {
     style-src 'self' 'unsafe-inline';
     font-src 'self';
     frame-src 'self' *.codesandbox.io aws.demdex.net;
-    img-src 'self' cm.everesttech.net amazonwebservices.d2.sc.omtrdc.net dpm.demdex.net https://images.unsplash.com;
+    img-src 'self' cm.everesttech.net amazonwebservices.d2.sc.omtrdc.net dpm.demdex.net https://images.unsplash.com data:;
     connect-src 'self' *.shortbread.aws.dev amazonwebservices.d2.sc.omtrdc.net dpm.demdex.net https://*.algolia.net https://*.algolianet.com;
-    script-src 'self' '${cspInlineScriptHash}' a0.awsstatic.com;
+    script-src 'unsafe-eval' 'self' '${cspInlineScriptHash}' a0.awsstatic.com;
```

Comment:
Why do we need to add this to prod? 

Is it safe? Is there an alternative way for `eval`?

Reply:
This was reviewed in an appsec meeting and it is ok. We need it because that is how react-live works (evaluating the code in the browser). The risk is low because there is no customer data on the site and it all runs in the browser so the only issue would be if the user themselves wrote malicious code that affected themself. We didn't need this previously because codesandbox was running in an iframe.